website/docs/r/dns_linked_zone.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "DNS Services"
+layout: "ibm"
+page_title: "IBM : dns_linked_zone"
+description: |-
+  Manages IBM Private DNS linked zone.
+---
+
+# ibm_dns_linked_zone
+
+The DNS linked zone resource allows users to request and manage linked zones. 
+
+
+## Example usage
+
+```
+resource "ibm_dns_linked_zone" "test" {
+  name          = "test_dns_linked_zone"
+  instance_id   = "****************************"
+  description   = "seczone terraform plugin test"
+  owner_instance_id = "**************************"
+  owner_zone_id = "************************"
+  label         = "test"
+}
+
+data "ibm_dns_linked_zone" "test-lz" {
+  depends_on  = [ibm_dns_linked_zone.test]
+  instance_id   = "***********************"
+}
+
+output "ibm_dns_linked_zone_id" {
+  value = data.ibm_dns_linked_zone.test-lz.instance_id
+}
+
+```
+
+## Argument reference
+Review the argument reference that you can specify for your resource. 
+
+- `instance_id` - (Required, String) The unique identifier of a service instance.
+- `description` - (Optional, String) Descriptive text of the linked zone.
+
+## Attribute reference
+In addition to all argument reference list, you can access the following attribute references after your resource is created. 
+
+- `created_on` - (Timestamp) The time (created On) of the Linked Zone. 
+- `modified_on` - (Timestamp) The time (modified On) of the Linked Zone.
+- `linked_zone_id` - (String) The unique ID of the DNS Services linked zone.
```

Comment:
IS this defined in schema of resource and the value is set in Read method

Reply:
Removed linked_zone_id from computed values